### ISSUE TITLE
Different hyper-parameter settings?

### ISSUE THREAD

**Issue by Wuziyi616** [ajhamdi/AdvPC#2]
Hi! Thank you for your great work. But I have a few questions about the argparse default parameters in attack.py. It's said in README that "The default parameters are the ones used in the paper", but I find some differences between the code and the paper:

1. The paper said you use "learning rate=0.01", but in the code [here](https://github.com/ajhamdi/AdvPC/blob/master/attack.py#L62) the lr is 0.005
2. The paper said you use "200 iterations of attack optimization" but it seems that you use 10-fold binary search as [here](https://github.com/ajhamdi/AdvPC/blob/master/attack.py#L94-L95), indicating that the default parameter is $10 \times 500 = 2500$ iterations in one attack
3. The paper said you use "2 different initialization for perturbation", but where is this hyper-parameter in the code?
4. The paper said you use "\kappa=30" for adv loss, but it seems that the default hyper-parameter is 0 [here](https://github.com/ajhamdi/AdvPC/blob/master/attack.py#L66-L68)
5. Also for the weight of some distance loss, the default parameters seems to be different

Actually I want to test my defense method on your attack, and I want to conduct fair comparison. So I hope you can modify the attack.py file and set the hyper-parameters you used in the paper to be the default ones. Thanks!

**Comment by ajhamdi** [comment]
The hyperparameters in the code are now adjusted to match the ones in the paper.

3. The parameter `step` controls how many different initializations ( binary search as well but since we are using constant weighting it stands for the initializations only ) .

5. The weights of the loss are either 1 or 0 depending on the type of attack  (hard:0 or soft:1 ) and the norm type ( L_infty, L_2, L_Chamfer, L_EMD ).
